tests/test_mappings/test_imports/test_modules/test_expense_custom_fields.py

```
@@ -8,7 +8,7 @@
 )
 
 from apps.workspaces.models import Workspace, FyleCredential
-from apps.mappings.imports.modules.expense_custom_fields import ExpenseCustomField
+from fyle_integrations_imports.modules.expense_custom_fields import ExpenseCustomField
```

Pylint (3.3.7)

[error] Unable to import 'fyle_integrations_imports.modules.expense_custom_fields'

(E0401)